pkg/util/config/feature_gate.go

```diff
@@ -37,6 +37,8 @@ const (
 	// values override the default set by allAlpha.
 	// e.g. allAlpha=false,newFeature=true will result in newFeature=true
 	allAlpha = "allAlpha"
```

Comment:
No objection if you want to rename. I don't think there are other consumers yet.